src/components/Button/Button.md

```diff
@@ -1,25 +1,25 @@
 ### Default Buttons
 ```jsx
-<Button onClick={() => { console.log('Hello, world!') }}>Primary</Button>{' '}
-<Button onClick={() => { console.log('Hello, world!') }} variant="success">Success</Button>{' '}
-<Button onClick={() => { console.log('Hello, world!') }} variant="danger">Danger</Button>{' '}
-<Button onClick={() => { console.log('Hello, world!') }} variant="plain">Plain</Button>{' '}
-<Button onClick={() => { console.log('Hello, world!') }} disabled>Disabled</Button>
+<Button margin="xs" onClick={() => { console.log('Click!') }}>Primary</Button>
```

Comment:
Will this look the same? I thought components delimited by newlines would run together without an explicit space.

Reply:
The margin adds the spacing between them. I specifically used `margin="xs"` here because it's  less verbose here than e.g `margin={{right: "xs"}}`.